test/unit/vl-mapactions-draw-action.test.js

```diff
@@ -55,10 +55,32 @@ describe('draw action', () => {
     const snappingLayer = new Vector({source: snappingSource});
     options.snapping = {
       layer: snappingLayer,
+      pixelTolerance: 1000,
     };
     drawAction = new VlDrawAction(layer, 'LineString', callback, options);
     const snapInteraction = drawAction.interactions.find((interaction) => interaction instanceof VlSnapInteraction);
     expect(snapInteraction.snapOptions.source).to.equal(snappingSource);
+    expect(snapInteraction.snapOptions.pixelTolerance).to.equal(1000);
+  });
+
+  it('als er een snapping layer is wordt die toegevoegd en verwijderd bij het aan- en afzetten van de actie', () => {
+    const snappingSource = new SourceVector({features: []});
+    const snappingLayer = new Vector({source: snappingSource});
+    const options = {
+      snapping: {
+        layer: snappingLayer,
+        pixelTolerance: 1000,
+      },
+    };
+    const drawAction = createDrawActionWithMap('Point', options);
+    sinon.stub(drawAction.map, 'addLayer');
+    drawAction.activate();
+    expect(drawAction.map.addLayer.called).to.be.true;
+    expect(drawAction.map.addLayer.getCall(0).args[0]).to.equal(snappingLayer);
```

Comment:
Eventueel `calledWith` overwegen i.p.v. `getCall(0).args[0]`, maar huidige code is zeker niet verkeerd.